FWCore/Framework/test/stubs/ToyIntProducers.cc

```diff
@@ -1016,6 +1039,7 @@ namespace edm::test {
       edm::EDPutTokenT<edmtest::IntProduct> token_;
       int value_;
     };
+    using IntTransformer = IntProducer;
```

Comment:
Best to add a comment explaining why this is different.

Reply:
Thanks, I added a comment.